test/index.ts

```diff
@@ -1389,3 +1389,122 @@ test('signedTypeMessage V4 with recursive types', (t) => {
     '0xf2ec61e636ff7bb3ac8bc2a4cc2c8b8f635dd1b2ec8094c963128b358e79c85c5ca6dd637ed7e80f0436fe8fce39c0e5f2082c9517fe677cc2917dcd6c84ba881c',
   );
 });
+
+test('unbound sign typed data utility functions', (t) => {
```

Comment:
I tested that these all failed on the `main` branch.

Reply:
will need to be converted to `jest` style to go along with this PR: https://github.com/MetaMask/eth-sig-util/pull/161

Reply:
Done!